addons/website_livechat/models/discuss_channel.py

```diff
@@ -58,8 +38,27 @@ def _field_store_repr(self, field_name):
 
     def _to_store_defaults(self, for_current_user=True):
         return super()._to_store_defaults(for_current_user=for_current_user) + [
+            Store.One(
+                "livechat_visitor_id",
+                [
+                    Store.One("country_id", ["code"]),
+                    "display_name",
+                    "history",
+                    "is_connected",
+                    Store.One("lang_id", ["name"]),
+                    "name",
+                    Store.One(
+                        "partner_id",
+                        [
+                            Store.One("country_id", ["code"], rename="country"),
+                            "display_name",
+                            "name",
```

Comment:
I think there's no point to return both, only the one used in priority should be returned.